modules/doc/content/newsletter/2022_02.md

```diff
@@ -74,6 +74,42 @@ to full compatibility with the current kernels.  Some features the Lagrangian sy
 - Axisymmetric and spherical coordinates.
 - Generalized plane strain and plane stress formulations.
 
+### Automatic differentiation support in MOOSE Functions
+
+Dual number support has been added to the MOOSE `Function` system. Two overloads
+now exist for the `value()` member function.
+
+```
+Real value(Real, Point);
+ADReal value(ADReal, ADPoint);
+```
+
+This enables evaluation of functions with dual numbers. As most legacy function
+do not implement  an AD overload of the `value()` function, the `Function` base
+class automatically provides one that uses the non-AD `value()`,
+`timeDerivative()`, and `gradient()` member functions to construct an AD result.
+Check out
+[`PiecewiseBilinear.C`](https://github.com/idaholab/moose/pull/20200/files#diff-269f04c2d048abf5058f746d123f609fa0f4982daa065de0ede74f2291f959be)
+to see how to update a function to support AD by using a templated
+`valueInternal()` function with virtual `value()` forwarders.
+
+Check out `BodyForce` and `CoupledValueFunctionMaterial` for applications of
+this new AD capability.
+
+Additionally the API for the `value()` and `timeDerivative()` members has been
+updated to permit the following usage:
+
+```
+auto r = func.value(t);
+auto r = func.value(t, x);
+auto r = func.value(t, x, y);
+auto r = func.value(t, x, y, z);
+```
+
+This simplifies code that uses function evaluations with custom variables, such
+as in `HeatConductionMaterial`, where a function can be supplied which only
+depends on time, and Temperature is supplied as the time argument.
```

Comment:
Don't fully understand. Does the function only depend on time or does temperature replace the time argument?

Reply:
what's called "time" internally in the function system is not necessarily the actual simulation time, but whatever the caller substituted in for the first argument. Yeah, I get it, it it confusing. If you have a suggestion on how to word it more clearly I'd be happy to adopt it.

Reply:
"where a function that only depends on temperature can be supplied. The variable temperature takes the place of the first argument (internally designated as ``time'')."?

If I understood that.